src/components/Button.js

```diff
@@ -1,18 +1,22 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useContext } from 'react';
 import { StyleSheet } from 'react-native';
 import { Button as RNEButton } from 'react-native-elements';
 
 import { colors, normalize } from '../config';
 import { DiagonalGradient } from './DiagonalGradient';
+import { OrientationContext } from '../OrientationProvider';
 
 export const Button = ({ title, onPress }) => {
+  const { orientation } = useContext(OrientationContext);
   return (
     <RNEButton
       onPress={onPress}
       title={title}
       titleStyle={styles.titleStyle}
-      containerStyle={styles.containerStyle}
+      containerStyle={
+        orientation === 'landscape' ? styles.containerStyleLandscape : styles.containerStyle
```

Comment:
to avoid duplication in styles, where `marginBottom: normalize(21)` is defined twice, one can use an array here.
with the array one can set multiple style objects, as well as setting them conditionally.
we use that in other places already. this means we can say:
- `styles.container` should always apply
- `styles.containerLandscape` should only apply on landscape
this results in the following code:
```
containerStyle={[
    styles.containerStyle,
    orientation === 'landscape' && styles.containerStyleLandscape
  ]}
```

this way, just `styles.container` needs the bottom margin, because it is applied every time.
i will update it.

Reply:
love it!!